org.eclipse.jdt.debug.ui/ui/org/eclipse/jdt/internal/debug/ui/console/JavaStackTraceHyperlink.java

```diff
@@ -190,8 +195,37 @@ public IStatus runInUIThread(IProgressMonitor monitor) {
 				} else if (source instanceof List) { // ambiguous
 					@SuppressWarnings("unchecked")
 					List<Object> matches = (List<Object>) source;
+					List<Object> exactMatchesFiltered = new ArrayList<>();
+					String originalHyperLink2 = originalHyperLink;
+					int firstMethodStartIndex = originalHyperLink2.indexOf("."); //$NON-NLS-1$
```

Comment:
to avoid `//$NON-NLS-1$` it's often possible to use single characters like `indexOf('.')`

Reply:
Okay